AirCasting/SearchAndFollow/CompleteScreen/Contollers/SearchAndFollowCompleteScreenService.swift

```diff
@@ -0,0 +1,63 @@
+// Created by Lunar on 04/05/2022.
+//
+
+import Foundation
+import Resolver
+
+protocol SearchAndFollowCompleteScreenService {
```

Comment:
changing name was the easy way out :D but let's say that's a service, so please rename or move the containing directory (it's called `Controllers`)